tests/e2e/yamcs/limits.e2e.spec.js

```diff
@@ -197,6 +199,92 @@ test.describe("Mdb runtime limits tests @yamcs", () => {
         // Ensure that the changed limits are now displayed without a reload
         await assertLimitLinesExistAndAreVisible(page);
     });
+
+    test('No new requests for data are made when limits are redrawn due to window resizing', async({ page }) => {
+        // Create an Overlay Plot
+        const overlayPlot = await createDomainObjectWithDefaults(page, {
+            type: 'Overlay Plot'
+        });
+
+        //Expand the myproject folder (/myproject)
```

Comment:
You can replace 209-220 with a direct navigation to the Detector_Temp XTCE. The test will become more resilient and significantly faster.

Reply:
OK

Reply:
Actually @unlikelyzero We need to drag and drop the Detector_Temp into the overlay plot. We can't skip these steps, right?